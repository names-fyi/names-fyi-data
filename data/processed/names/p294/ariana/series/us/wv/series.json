[{"y":1997,"t":6,"m":0,"f":6,"mr":null,"fr":249,"mp":0,"fp":0.0008179959100204499},{"y":2000,"t":7,"m":0,"f":7,"mr":null,"fr":234,"mp":0,"fp":0.0009627286480539128},{"y":2001,"t":8,"m":0,"f":8,"mr":null,"fr":206,"mp":0,"fp":0.001120919153706039},{"y":2002,"t":6,"m":0,"f":6,"mr":null,"fr":269,"mp":0,"fp":0.0008405715886803026},{"y":2003,"t":5,"m":0,"f":5,"mr":null,"fr":312,"mp":0,"fp":0.000693000693000693},{"y":2004,"t":10,"m":0,"f":10,"mr":null,"fr":178,"mp":0,"fp":0.0013931457230426303},{"y":2005,"t":9,"m":0,"f":9,"mr":null,"fr":196,"mp":0,"fp":0.0013109978150036416},{"y":2006,"t":10,"m":0,"f":10,"mr":null,"fr":178,"mp":0,"fp":0.0014271442842871413},{"y":2007,"t":13,"m":0,"f":13,"mr":null,"fr":149,"mp":0,"fp":0.001849217638691323},{"y":2008,"t":8,"m":0,"f":8,"mr":null,"fr":226,"mp":0,"fp":0.0011644832605531296},{"y":2009,"t":17,"m":0,"f":17,"mr":null,"fr":109,"mp":0,"fp":0.0025118203309692673},{"y":2010,"t":12,"m":0,"f":12,"mr":null,"fr":142,"mp":0,"fp":0.0018214936247723133},{"y":2011,"t":13,"m":0,"f":13,"mr":null,"fr":140,"mp":0,"fp":0.0019880715705765406},{"y":2012,"t":6,"m":0,"f":6,"mr":null,"fr":283,"mp":0,"fp":0.0009113001215066828},{"y":2013,"t":22,"m":0,"f":22,"mr":null,"fr":77,"mp":0,"fp":0.003327283726557774},{"y":2014,"t":18,"m":0,"f":18,"mr":null,"fr":99,"mp":0,"fp":0.0028535193405199747},{"y":2015,"t":17,"m":0,"f":17,"mr":null,"fr":101,"mp":0,"fp":0.002696272799365583},{"y":2016,"t":20,"m":0,"f":20,"mr":null,"fr":81,"mp":0,"fp":0.003287310979618672},{"y":2017,"t":13,"m":0,"f":13,"mr":null,"fr":123,"mp":0,"fp":0.002226027397260274},{"y":2018,"t":9,"m":0,"f":9,"mr":null,"fr":190,"mp":0,"fp":0.0016068559185859668},{"y":2019,"t":10,"m":0,"f":10,"mr":null,"fr":168,"mp":0,"fp":0.0017642907551164433},{"y":2022,"t":6,"m":0,"f":6,"mr":null,"fr":274,"mp":0,"fp":0.001129305477131564}]