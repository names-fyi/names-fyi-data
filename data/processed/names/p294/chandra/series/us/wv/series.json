[{"y":1970,"t":5,"m":0,"f":5,"mr":null,"fr":336,"mp":0,"fp":0.00039053346871826915},{"y":1975,"t":5,"m":0,"f":5,"mr":null,"fr":328,"mp":0,"fp":0.0004464285714285714},{"y":1977,"t":15,"m":0,"f":15,"mr":null,"fr":173,"mp":0,"fp":0.0012164463547157571},{"y":1978,"t":8,"m":0,"f":8,"mr":null,"fr":253,"mp":0,"fp":0.0006778512116590408},{"y":1979,"t":6,"m":0,"f":6,"mr":null,"fr":306,"mp":0,"fp":0.0004926108374384236},{"y":1981,"t":9,"m":0,"f":9,"mr":null,"fr":224,"mp":0,"fp":0.0007808433107756377},{"y":1983,"t":5,"m":0,"f":5,"mr":null,"fr":314,"mp":0,"fp":0.00046720239207624745},{"y":1984,"t":6,"m":0,"f":6,"mr":null,"fr":266,"mp":0,"fp":0.0006009615384615385},{"y":1985,"t":5,"m":0,"f":5,"mr":null,"fr":307,"mp":0,"fp":0.0005088540606554041},{"y":1986,"t":5,"m":0,"f":5,"mr":null,"fr":291,"mp":0,"fp":0.0005294366793731469},{"y":1988,"t":5,"m":0,"f":5,"mr":null,"fr":275,"mp":0,"fp":0.0005772338951743247},{"y":1989,"t":6,"m":0,"f":6,"mr":null,"fr":251,"mp":0,"fp":0.0006822057987492893},{"y":1991,"t":5,"m":0,"f":5,"mr":null,"fr":291,"mp":0,"fp":0.0005704506560182544},{"y":1993,"t":6,"m":0,"f":6,"mr":null,"fr":259,"mp":0,"fp":0.0007122507122507123}]