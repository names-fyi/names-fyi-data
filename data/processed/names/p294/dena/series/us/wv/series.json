[{"y":1930,"t":5,"m":0,"f":5,"mr":null,"fr":390,"mp":0,"fp":0.00027221254355400695},{"y":1952,"t":5,"m":0,"f":5,"mr":null,"fr":408,"mp":0,"fp":0.00022514409221902016},{"y":1953,"t":6,"m":0,"f":6,"mr":null,"fr":367,"mp":0,"fp":0.0002882536632236368},{"y":1955,"t":6,"m":0,"f":6,"mr":null,"fr":364,"mp":0,"fp":0.0003052037234854265},{"y":1956,"t":8,"m":0,"f":8,"mr":null,"fr":316,"mp":0,"fp":0.0004019090680733484},{"y":1957,"t":8,"m":0,"f":8,"mr":null,"fr":314,"mp":0,"fp":0.0004048992813037757},{"y":1958,"t":6,"m":0,"f":6,"mr":null,"fr":376,"mp":0,"fp":0.00031036623215394165},{"y":1959,"t":8,"m":0,"f":8,"mr":null,"fr":306,"mp":0,"fp":0.00043660972548163513},{"y":1960,"t":5,"m":0,"f":5,"mr":null,"fr":396,"mp":0,"fp":0.00028740587457607634},{"y":1961,"t":5,"m":0,"f":5,"mr":null,"fr":397,"mp":0,"fp":0.0002974773917182294},{"y":1962,"t":11,"m":0,"f":11,"mr":null,"fr":243,"mp":0,"fp":0.0006820436507936508},{"y":1963,"t":8,"m":0,"f":8,"mr":null,"fr":287,"mp":0,"fp":0.0005286460054186215},{"y":1964,"t":8,"m":0,"f":8,"mr":null,"fr":288,"mp":0,"fp":0.0005249688299757202},{"y":1965,"t":6,"m":0,"f":6,"mr":null,"fr":311,"mp":0,"fp":0.00044264109184802655},{"y":1967,"t":10,"m":0,"f":10,"mr":null,"fr":217,"mp":0,"fp":0.0008111615833874108},{"y":1968,"t":8,"m":0,"f":8,"mr":null,"fr":248,"mp":0,"fp":0.0006563294773976536},{"y":1969,"t":9,"m":0,"f":9,"mr":null,"fr":242,"mp":0,"fp":0.0007580862533692722},{"y":1970,"t":11,"m":0,"f":11,"mr":null,"fr":223,"mp":0,"fp":0.0008591736311801922},{"y":1971,"t":12,"m":0,"f":12,"mr":null,"fr":213,"mp":0,"fp":0.0009208103130755065},{"y":1972,"t":5,"m":0,"f":5,"mr":null,"fr":349,"mp":0,"fp":0.00041037426132632963},{"y":1973,"t":8,"m":0,"f":8,"mr":null,"fr":261,"mp":0,"fp":0.0007028024246683651},{"y":1975,"t":7,"m":0,"f":7,"mr":null,"fr":271,"mp":0,"fp":0.000625},{"y":1976,"t":9,"m":0,"f":9,"mr":null,"fr":236,"mp":0,"fp":0.0007768666378938282}]