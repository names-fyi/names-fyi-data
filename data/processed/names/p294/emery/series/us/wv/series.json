[{"y":1912,"t":6,"m":6,"f":0,"mr":156,"fr":null,"mp":0.0012036108324974925,"fp":0},{"y":1913,"t":9,"m":9,"f":0,"mr":128,"fr":null,"mp":0.0014885874958650348,"fp":0},{"y":1915,"t":11,"m":11,"f":0,"mr":164,"fr":null,"mp":0.0009887640449438202,"fp":0},{"y":1916,"t":13,"m":13,"f":0,"mr":148,"fr":null,"mp":0.0011626867006528934,"fp":0},{"y":1917,"t":8,"m":8,"f":0,"mr":212,"fr":null,"mp":0.0006645070188553867,"fp":0},{"y":1918,"t":14,"m":14,"f":0,"mr":162,"fr":null,"mp":0.0009984310369419483,"fp":0},{"y":1919,"t":12,"m":12,"f":0,"mr":181,"fr":null,"mp":0.0008969950665271341,"fp":0},{"y":1920,"t":10,"m":10,"f":0,"mr":215,"fr":null,"mp":0.0006576351440220965,"fp":0},{"y":1921,"t":9,"m":9,"f":0,"mr":250,"fr":null,"mp":0.0005348229141906347,"fp":0},{"y":1922,"t":10,"m":10,"f":0,"mr":229,"fr":null,"mp":0.0006004563468235859,"fp":0},{"y":1923,"t":7,"m":7,"f":0,"mr":297,"fr":null,"mp":0.0004065748969042226,"fp":0},{"y":1924,"t":12,"m":12,"f":0,"mr":212,"fr":null,"mp":0.0006501246072163832,"fp":0},{"y":1925,"t":6,"m":6,"f":0,"mr":333,"fr":null,"mp":0.00034050280914817545,"fp":0},{"y":1926,"t":7,"m":7,"f":0,"mr":293,"fr":null,"mp":0.0003987467957846767,"fp":0},{"y":1927,"t":11,"m":11,"f":0,"mr":224,"fr":null,"mp":0.0006016189017720411,"fp":0},{"y":1928,"t":6,"m":6,"f":0,"mr":310,"fr":null,"mp":0.00033385265969285556,"fp":0},{"y":1929,"t":6,"m":6,"f":0,"mr":318,"fr":null,"mp":0.00035859431030360985,"fp":0},{"y":1930,"t":8,"m":8,"f":0,"mr":253,"fr":null,"mp":0.00046658112679342123,"fp":0},{"y":1931,"t":8,"m":8,"f":0,"mr":261,"fr":null,"mp":0.0004938271604938272,"fp":0},{"y":1934,"t":10,"m":10,"f":0,"mr":229,"fr":null,"mp":0.0005600044800358403,"fp":0},{"y":1935,"t":9,"m":9,"f":0,"mr":240,"fr":null,"mp":0.0005022041180737682,"fp":0},{"y":1936,"t":6,"m":6,"f":0,"mr":306,"fr":null,"mp":0.00033396415451408216,"fp":0},{"y":1938,"t":11,"m":11,"f":0,"mr":220,"fr":null,"mp":0.0005738431843079972,"fp":0},{"y":1939,"t":12,"m":12,"f":0,"mr":211,"fr":null,"mp":0.0006326444538169548,"fp":0},{"y":1941,"t":5,"m":5,"f":0,"mr":336,"fr":null,"mp":0.0002518257365902795,"fp":0},{"y":1944,"t":6,"m":6,"f":0,"mr":295,"fr":null,"mp":0.00031794817444756504,"fp":0},{"y":1945,"t":5,"m":5,"f":0,"mr":327,"fr":null,"mp":0.0002735229759299781,"fp":0},{"y":1947,"t":6,"m":6,"f":0,"mr":343,"fr":null,"mp":0.00022904260192395785,"fp":0},{"y":1951,"t":8,"m":8,"f":0,"mr":278,"fr":null,"mp":0.0003285016219767585,"fp":0},{"y":1960,"t":5,"m":5,"f":0,"mr":311,"fr":null,"mp":0.0002636296530633766,"fp":0},{"y":1962,"t":6,"m":6,"f":0,"mr":274,"fr":null,"mp":0.0003390596745027125,"fp":0},{"y":1963,"t":5,"m":5,"f":0,"mr":304,"fr":null,"mp":0.00029401387745501585,"fp":0},{"y":1965,"t":5,"m":5,"f":0,"mr":283,"fr":null,"mp":0.0003322259136212625,"fp":0},{"y":2012,"t":5,"m":0,"f":5,"mr":null,"fr":347,"mp":0,"fp":0.0007594167679222357},{"y":2014,"t":12,"m":0,"f":12,"mr":null,"fr":156,"mp":0,"fp":0.0019023462270133164},{"y":2015,"t":10,"m":0,"f":10,"mr":null,"fr":171,"mp":0,"fp":0.0015860428231562252},{"y":2016,"t":11,"m":0,"f":11,"mr":null,"fr":170,"mp":0,"fp":0.0018080210387902695},{"y":2017,"t":14,"m":0,"f":14,"mr":null,"fr":112,"mp":0,"fp":0.002397260273972603},{"y":2018,"t":11,"m":0,"f":11,"mr":null,"fr":169,"mp":0,"fp":0.0019639350116050706},{"y":2019,"t":11,"m":0,"f":11,"mr":null,"fr":156,"mp":0,"fp":0.0019407198306280876},{"y":2020,"t":18,"m":0,"f":18,"mr":null,"fr":85,"mp":0,"fp":0.0033247137052087182},{"y":2021,"t":21,"m":0,"f":21,"mr":null,"fr":63,"mp":0,"fp":0.003865267807840972},{"y":2022,"t":18,"m":0,"f":18,"mr":null,"fr":81,"mp":0,"fp":0.0033879164313946925}]