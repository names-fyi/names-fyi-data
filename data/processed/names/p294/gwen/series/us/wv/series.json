[{"y":1948,"t":5,"m":0,"f":5,"mr":null,"fr":416,"mp":0,"fp":0.00020889910173386254},{"y":1949,"t":6,"m":0,"f":6,"mr":null,"fr":374,"mp":0,"fp":0.00025157232704402514},{"y":1956,"t":8,"m":0,"f":8,"mr":null,"fr":323,"mp":0,"fp":0.0004019090680733484},{"y":1957,"t":10,"m":0,"f":10,"mr":null,"fr":279,"mp":0,"fp":0.0005061241016297196},{"y":1959,"t":6,"m":0,"f":6,"mr":null,"fr":367,"mp":0,"fp":0.00032745729411122633}]