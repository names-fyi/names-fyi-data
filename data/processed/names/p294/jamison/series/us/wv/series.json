[{"y":1973,"t":8,"m":8,"f":0,"mr":192,"fr":null,"mp":0.0006134028523232633,"fp":0},{"y":1976,"t":8,"m":8,"f":0,"mr":197,"fr":null,"mp":0.0006051894999621756,"fp":0},{"y":2001,"t":6,"m":6,"f":0,"mr":237,"fr":null,"mp":0.0006855575868372943,"fp":0},{"y":2006,"t":6,"m":6,"f":0,"mr":253,"fr":null,"mp":0.0006846188954815153,"fp":0},{"y":2007,"t":6,"m":6,"f":0,"mr":270,"fr":null,"mp":0.0006659267480577136,"fp":0},{"y":2014,"t":5,"m":5,"f":0,"mr":348,"fr":null,"mp":0.0006184291898577613,"fp":0},{"y":2015,"t":6,"m":6,"f":0,"mr":308,"fr":null,"mp":0.000737372496005899,"fp":0},{"y":2016,"t":5,"m":5,"f":0,"mr":350,"fr":null,"mp":0.0006417661404184315,"fp":0},{"y":2017,"t":10,"m":10,"f":0,"mr":213,"fr":null,"mp":0.001341741580571582,"fp":0},{"y":2018,"t":11,"m":11,"f":0,"mr":206,"fr":null,"mp":0.001465299054216065,"fp":0},{"y":2019,"t":18,"m":18,"f":0,"mr":126,"fr":null,"mp":0.0024667671645881867,"fp":0},{"y":2020,"t":7,"m":7,"f":0,"mr":288,"fr":null,"mp":0.0010127314814814814,"fp":0},{"y":2021,"t":5,"m":5,"f":0,"mr":361,"fr":null,"mp":0.0007185972980741593,"fp":0},{"y":2022,"t":8,"m":8,"f":0,"mr":252,"fr":null,"mp":0.0012108369910700772,"fp":0}]