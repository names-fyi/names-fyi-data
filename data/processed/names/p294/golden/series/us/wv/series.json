[{"y":1912,"t":8,"m":8,"f":0,"mr":127,"fr":null,"mp":0.0016048144433299899,"fp":0},{"y":1914,"t":6,"m":6,"f":0,"mr":217,"fr":null,"mp":0.00074673304293715,"fp":0},{"y":1915,"t":8,"m":8,"f":0,"mr":213,"fr":null,"mp":0.0007191011235955057,"fp":0},{"y":1916,"t":5,"m":5,"f":0,"mr":306,"fr":null,"mp":0.0004471871925588051,"fp":0},{"y":1918,"t":9,"m":9,"f":0,"mr":226,"fr":null,"mp":0.0006418485237483953,"fp":0},{"y":1919,"t":7,"m":7,"f":0,"mr":263,"fr":null,"mp":0.0005232471221408282,"fp":0},{"y":1920,"t":11,"m":11,"f":0,"mr":198,"fr":null,"mp":0.0007233986584243062,"fp":0},{"y":1921,"t":13,"m":13,"f":0,"mr":189,"fr":null,"mp":0.0007725219871642501,"fp":0},{"y":1922,"t":7,"m":7,"f":0,"mr":294,"fr":null,"mp":0.00042031944277651017,"fp":0},{"y":1923,"t":6,"m":6,"f":0,"mr":328,"fr":null,"mp":0.0003484927687750479,"fp":0},{"y":1924,"t":10,"m":10,"f":0,"mr":240,"fr":null,"mp":0.0005417705060136526,"fp":0},{"y":1925,"t":8,"m":8,"f":0,"mr":276,"fr":null,"mp":0.00045400374553090066,"fp":0},{"y":1927,"t":6,"m":6,"f":0,"mr":326,"fr":null,"mp":0.00032815576460293154,"fp":0},{"y":1928,"t":8,"m":8,"f":0,"mr":272,"fr":null,"mp":0.0004451368795904741,"fp":0},{"y":1929,"t":5,"m":5,"f":0,"mr":352,"fr":null,"mp":0.00029882859191967485,"fp":0},{"y":1930,"t":6,"m":6,"f":0,"mr":305,"fr":null,"mp":0.0003499358450950659,"fp":0},{"y":1931,"t":7,"m":7,"f":0,"mr":283,"fr":null,"mp":0.0004320987654320988,"fp":0},{"y":1935,"t":7,"m":7,"f":0,"mr":277,"fr":null,"mp":0.0003906032029462642,"fp":0},{"y":1939,"t":5,"m":5,"f":0,"mr":345,"fr":null,"mp":0.0002636018557570645,"fp":0},{"y":1945,"t":5,"m":5,"f":0,"mr":330,"fr":null,"mp":0.0002735229759299781,"fp":0},{"y":1946,"t":6,"m":6,"f":0,"mr":313,"fr":null,"mp":0.00025868759161852203,"fp":0}]